date-input.html

```diff
@@ -53,40 +53,85 @@
     <span aria-hidden id="monthLabel" hidden>Month</span>
     <span aria-hidden id="yearLabel" hidden>Year</span>
 
-    <div class="container">
-      <input is="iron-input"
-          id="expirationMonth"
-          aria-labelledby$="[[_computeAriaLabel(ariaLabelPrefix,'monthLabel')]]"
-          required$="[[required]]"
-          maxlength="2"
+    <span id="template-placeholder"></span>
+
+    <template id="v1">
+      <div class="container">
+        <input is="iron-input"
+            id="expirationMonth"
+            aria-labelledby$="[[_computeAriaLabel(ariaLabelPrefix,'monthLabel')]]"
+            required$="[[required]]"
+            maxlength="2"
+            bind-value="{{month}}"
+            placeholder="MM"
+            allowed-pattern="[0-9]"
+            prevent-invalid-input
+            autocomplete="cc-exp-month"
+            type="tel"
+            disabled$="[[disabled]]"
+            invalid="{{invalid}}"
+            autofocus$="[[autofocus]]"
+            inputmode$="[[inputmode]]"
+            readonly$="[[readonly]]">
+        <span>/</span>
+        <input is="iron-input"
+            id="expirationYear"
+            aria-labelledby$="[[_computeAriaLabel(ariaLabelPrefix,'yearLabel')]]"
+            required$="[[required]]"
+            maxlength="2"
+            bind-value="{{year}}"
+            placeholder="YY"
+            allowed-pattern="[0-9]"
+            prevent-invalid-input
+            autocomplete="cc-exp-year"
+            type="tel"
+            disabled$="[[disabled]]"
+            invalid="{{invalid}}"
+            inputmode$="[[inputmode]]"
+            readonly$="[[readonly]]">
+      </div>
+    </template>
+
+    <template id="v2">
+
+      <iron-input id="expirationMonth" slot="monthInput"
```

Comment:
the v2 template is missing the `<div class="container">`... maybe that container should be pulled outside of the version-specific templates and into the main template?